test/tools/yulrun.cpp

```diff
@@ -120,7 +120,8 @@ Allowed options)",
 		po::options_description::m_default_line_length - 23);
 	options.add_options()
 		("help", "Show this help screen.");
-	options.add_options()("input-file", po::value<vector<string>>(), "input file");
+		("help", "Show this help screen.")
```

Comment:
well, I didn't know how write a removal of a line in the suggestion, but I meant to replace line 122 ;)